TealiumContentsquare.xcodeproj/project.pbxproj

```diff
@@ -66,7 +57,8 @@
 			files = (
 				15430459277C6820004FF14A /* TealiumCore.xcframework in Frameworks */,
 				1543045A277C6820004FF14A /* TealiumRemoteCommands.xcframework in Frameworks */,
-				15430458277C6820004FF14A /* SwiftProtobuf.xcframework in Frameworks */,
+				9E6990102DDB423A0005BB8B /* CSSwiftProtobuf.xcframework in Frameworks */,
```

Comment:
We added `CSSwiftProtobuf` but didn't remove `SwiftProtobuf` from everywhere in the project. 
Some references are left, leading to a compile error in the project due to it being in the Frameworks list when you open the project.